package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "glossary-converter": "node tools/glossary-converter.js",
-    "postinstall-mv2": "cd extension-manifest-v2 && npm run licenses",
+    "postinstall-mv2": "cd extension-manifest-v2 && npm run licenses && npm run download-trackerdb",
```

Comment:
Is there a reason why packages postinstall scripts are not in the packages, but here? It was before like `npm postinstall -ws ...`.

Reply:
Running `postinstall` from packages is not a supported usecase for npm workspaces. it started to fail for me with latest npm upgrade (when we've moved to node 19) - 

This approach is more predictable

Reply:
Ok, understand, but it ties the root with package internals... I get the reason but still don't like the solution.